inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/FieldInfoUtils.java

```diff
@@ -87,10 +87,11 @@
             FieldInfo sinkField = getFieldInfo(field.getFieldName(), field.getFieldType(),
                     field.getIsMetaField() == 1, field.getFieldFormat());
             sinkFields.add(sinkField);
-
-            FieldInfo sourceField = getFieldInfo(field.getSourceFieldName(),
-                    field.getSourceFieldType(), field.getIsMetaField() == 1, field.getFieldFormat());
-            mappingUnitList.add(new FieldMappingUnit(sourceField, sinkField));
+            if (StringUtils.isNotBlank(field.getSourceFieldName())) {
```

Comment:
The source and sink fields cannot be empty here. If they are empty, it means that there is a problem with the configuration, and an exception should be thrown.

Reply:
Perhaps we should do non-null checks when saving and modifying source and sink fields.

Reply:
Writing a data source like hive requires all table field information, but there are some field mapping scenarios, such as 8 fields on the user source side and 10 fields on the target side. At this time, all the fields on the target side need to be passed in. But only part of the field mapping can be passed in.

Reply:
OK, then the Sort module needs to handle this scenario.